SCAutolib/cli_commands.py

```diff
@@ -154,6 +154,15 @@ def setup_user(ctx, name, card_dir, card_type, passwd, pin, user_type):
     exit(ReturnCode.SUCCESS.value)
 
 
+@click.command()
+@click.pass_context
+def cleanup(ctx):
+    """Configure entire system for smart cards based on the config file."""
```

Comment:
is this doc text up to date? This does not sound like some copy&paste from other command.